### PR TITLE
cal: implement columns/landscape mode

### PR DESCRIPTION
**Pull request by t-8ch**
Replaces #117

### CODE REVIEW

**Comment by t-8ch** [comment]
@soulseekah Does this work for you?
You can do `cal --year --column 4` to get a landscape version.

**Comment by soulseekah** [comment]
> @soulseekah Does this work for you? You can do `cal --year --column 4` to get a landscape version.

I love it! Thanks.

File: misc-utils/cal.1.adoc [hunk 1]
```diff
@@ -117,6 +117,9 @@ Display week numbers in the calendar (US or ISO-8601). See the *NOTES* section f
 *--color*[=_when_]::
 Colorize the output. The optional argument _when_ can be *auto*, *never* or *always*. If the _when_ argument is omitted, it defaults to *auto*. The colors can be disabled; for the current built-in default see the *--help* output. See also the *COLORS* section.
 
+*-c, --columns*[=_columns_]::
```
Comment (karelzak) [inline misc-utils/cal.1.adoc:120]: It seems like the argument is optional (due to [ ] ), but getopt_long() defines it as required.

Comment (karelzak) [reply misc-utils/cal.1.adoc:120]: Fixed.

**Comment by karelzak** [comment]
@t-8ch thanks!

**Pull request merged**